webapp/src/main/resources/db/migration/V44__Add_imported.sql

```diff
@@ -0,0 +1,2 @@
+alter table `drop` add column partiallyImported bit not null default false;
```

Comment:
same as in the entity `partially_imported`

Reply:
Done